packages/client/wallets/smart-wallet/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { blockchainToChainId } from "@crossmint/common-sdk-base";
+export { blockchainToChainId, EVMBlockchainIncludingTestnet as Chain } from "@crossmint/common-sdk-base";
```

Comment:
I think `Chain` is better than `Blockchain` here. fwiw it's how we've named things in `crossbit-main` to